File: php/control_flow/for_loop/for_loop_formatting.php
<?php
/*
 * Copyright (C) 2019-2026 Katarzyna Krasińska
 * Languagium - https://github.com/katheroine/languagium
 * Licensed under GPL-3.0 - see LICENSE.md
 */

for ($i = 0; $i < 10; $i++)
  echo "{$i}...\n";

echo "\n";

for ($i = 0; $i < 10; $i++) echo "{$i}...\n";

echo "\n";

for ($i = 0; $i < 10; $i++) {
  echo "{$i}...\n";
}

echo "\n";

// Shortened form for HTML templates:

for ($i = 0; $i < 10; $i++):
  echo "{$i}...\n";
endfor;

echo "\n";

for ($i = 0; $i < 10; $i++): echo "{$i}...\n"; endfor;

echo "\n";

File: c++/control_flow/for_loop/for_loop_formatting.cpp
/*
 * Copyright (C) 2019-2026 Katarzyna Krasińska
 * Languagium - https://github.com/katheroine/languagium
 * Licensed under GPL-3.0 - see LICENSE.md
 */
// SPDX-License-Identifier: GPL-3.0-or-later

#include <iostream>

int main()
{
  int i;

  for (i = 0; i < 10; i++)
    std::cout << i << "...\n";

  std::cout << std::endl;

  for (int i = 0; i < 10; i++)
    std::cout << i << "...\n";

  std::cout << std::endl;

  for (int i = 0; i < 10; i++) std::cout << i << "...\n";

  std::cout << std::endl;

  for (int i = 0; i < 10; i++)
  {
    std::cout << i << "...\n";
  }

  std::cout << std::endl;
}
